priv/templates/extended_bin

```diff
@@ -343,6 +343,24 @@ case "$1" in
              "unpack" "$REL_NAME" "$NAME_TYPE" "$NAME" "$COOKIE" "$2"
         ;;
 
+    uninstall)
```

Comment:
Could the uninstall command be a part of `upgrade|downgrade|install)` ?
it seems very similar to those..